patches/tModLoader/Terraria/release_extras/tMLMod.targets

```diff
@@ -39,7 +39,12 @@
 	</ItemGroup>
 
 	<Target Name="BuildMod" AfterTargets="Build">
-		<Exec Command="dotnet $(tMLServerPath) -build $(ProjectDir) -eac $(TargetPath) -define &quot;$(DefineConstants)&quot; -unsafe $(AllowUnsafeBlocks) $(ExtraBuildModFlags)" WorkingDirectory="$(tMLSteamPath)"/>
+		<Exec Command="dotnet $(tMLServerPath) -build $(ProjectDir) -eac $(TargetPath) -define &quot;$(DefineConstants)&quot; -unsafe $(AllowUnsafeBlocks) $(ExtraBuildModFlags)" WorkingDirectory="$(tMLSteamPath)" ConsoleToMSBuild="true" ContinueOnError="ErrorAndContinue">
+			<Output TaskParameter="ConsoleOutput" PropertyName="ModBuildOutput"/>
+			<Output TaskParameter="ExitCode" PropertyName="ModBuildExitCode"/>
+		</Exec>
+		<Error Text="Please close tModLoader or disable the mod in-game to build mods directly" Condition=" '$(ModBuildExitCode)' == '1001' "/>
+		<Error Text="Build failed, see output panel for details" Condition=" '$(ModBuildExitCode)' == '1' "/>
```

Comment:
Consider using `!= 0` instead of `== 1`. If it causes both error messages to show, then consider using the `<Choose><When></When><Otherwise></Otherwise></Choose>` tags

Reply:
That works. Error Tasks seem to work in order, stopping at the first one that fails, so it only shows 1.